### DISCUSSION TITLE
NFT projects returns an empty NFT

### DISCUSSION THREAD

**Discussion by Marvelousy20** [Cyfrin/foundry-full-course-cu#2708] [Q&A]
![Screenshot 2024-10-19 at 1 23 06 AM](https://github.com/user-attachments/assets/6238ea20-0713-4031-8544-28467cc42679)

After I had completed the MoodNft project, I got the above (attached). The NFT svg was blank. Same happened for the Basic Nft as well.
I'd love to ask if anyone is facing the same issue and how can I resolve this? Thanks.

**Comment by EngrPips** [comment]
Hello @Marvelousy20, Please read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2112 as it may help you
